lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -1114,6 +1114,7 @@ def install_puppet_agent_pe_promoted_repo_on( hosts, opts )
               if arch == 'x86_64'
                 arch = 'amd64'
               end
+              version = version[0,2] + '.' + version[2,2] if variant =~ /ubuntu/
```

Comment:
@anodelman do we already have a way of solving this issue?  I feel like it's something we've probably seen before, but I'm not recalling if we have a better solution for it..